### PR TITLE
Occultist Addition: Occultist weapon sprites and partial stats.

### PR DESCRIPTION
**Pull request by Jamini**
## About The Pull Request

Sprites, Partial Functionality, and Backend support for Occultist weapons

The weapons are as follows:

Non-Eluclidiean Blade - Causes hallucinations and direclty damages sanity
Soulcrusher - Causes agony and directly damages sanity. Sparks on attack.
Chainripper - Deals more damage than other occultist weapons. Greatly damages sanity.

Moonrise - Occultist Laser Rifle, deal sanity and burn damage
Eclipse - Occultist Projectile Rifle, based on the sol, deals brute and sanity damage
Muramasen - Occultist plasma rifle, based on the cassad, deals burn and sanity damage

## Why It's Good For The Game

Occultist framework things

## Changelog
```changelog
add: Sprites, functionality for a lot of Occultist weapons
fix: Psy damage actually works now
fix: Fixed some mercenary things that got wonkied and were preventing compile

```

### CODE REVIEW

File: zzzz_modular_occulus/code/game/gamemodes/occultist/occultist_weapons.dm [hunk 1]
```diff
@@ -1,17 +1,173 @@
...
+	..()
+	return
+
+
+/obj/item/weapon/tool/hammer/homewrecker/cult
+	name = "Soulcrusher"
+
+	icon = 'zzzz_modular_occulus/icons/obj/occultist_weap/occhammer.dmi'
+
+	desc = "Heavy extremely sledgehammer. A Handle resembles a chunk welded to which a large long steel."
```
Comment (ShadzFen) [inline zzzz_modular_occulus/code/game/gamemodes/occultist/occultist_weapons.dm:47]: Heavy extremely sledgehammer indeed, however you aren't Peter Gabriel. Is this a typo?

Comment (Jamini) [reply zzzz_modular_occulus/code/game/gamemodes/occultist/occultist_weapons.dm:47]: No. The descriptions are intentionally fucked up.

**Review by ShadzFen — APPROVED** [review]
Looks fine. Modular and properly blacklisted. Cool to see this framework coming along nicely.

**Comment by Jamini** [comment]
> Looks fine. Modular and properly blacklisted. Cool to see this framework coming along nicely.

Are awesome. Merging now you.

**Pull request merged**